### DISCUSSION TITLE
Difference between Global and Average results for Reverberation Time

### DISCUSSION THREAD

**Discussion by user11042023** [Ifsttar/I-Simpa#358] [Q&A]
Hi to everyone,

I have some doubts about the Global and Average results calculated for the RT.

I noticed that calculating the RT value in the acoustics parameters, it gives two different values: the global and the average.
While the average is simply the linear average of all the RT corresponding to each frequency, I couldn't find any specific formula corresponding to the result of the global value. I noticed that the global is always higher than the average RT value, and while the average doesn't correspond to any plane receiver map, the global value is often corresponding to that map. 

The only thing that I found is that the manual mentions two kinds of averages: the linear (useful for: G, T, EDT, C, D, ts, LF, LFC) and the energetic (useful for LJ).

My questions about the global RT value are:
- what is the corresponding formula to get that result for the global value?
- I understand if it is the result of the simulation but then I still don't understand which value (average or global) I should take into consideration.

Thanks a lot for your time

![Immagine 2023-04-11 114845](https://user-images.githubusercontent.com/130450611/231123759-1832b4e0-24fb-47ca-af94-c319fc230fd8.png)

**Comment by nicolas-f** [comment]
Hi,


>     * what is the corresponding formula to get that result for the global value?
> 
You can't compute from this table the global level because you need the energetic Sum of frequencies over time, then compute the Global Sound level decrease of sound level in order to compute the global reverberation time.

https://i-simpa-wiki.readthedocs.io/fr/latest/room_acoustics_parameters.html#reverberation-time-t

>     * I understand if it is the result of the simulation but then I still don't understand which value (average or global) I should take into consideration.

The global RT is the value you have to use.

**Comment by user11042023** [comment]
Thanks a lot for your answer. 

Just to understand, if the global RT is the important value, what is the meaning of calculating the average of RT value?

Thanks again.
